internal/rpchelp/helpdescs_en_US.go

```diff
@@ -348,6 +348,29 @@ var helpDescsEnUS = map[string]string{
 	"help--result0":    "List of commands",
 	"help--result1":    "Help for specified command",
 
+	// GetTxOutCmd help.
+	"gettxout--synopsis":      "Returns information about an unspent transaction output.",
+	"gettxout-txid":           "The hash of the transaction",
+	"gettxout-vout":           "The index of the output",
+	"gettxout-tree":           "The tree of the transaction",
+	"gettxout-includemempool": "Include the mempool when true",
+
+	// GetTxOutResult help.
+	"gettxoutresult-bestblock":     "The block hash that contains the transaction output",
+	"gettxoutresult-confirmations": "The number of confirmations",
+	"gettxoutresult-value":         "The transaction amount in DCR",
+	"gettxoutresult-scriptPubKey":  "The public key script used to pay coins as a JSON object",
+	"gettxoutresult-version":       "The transaction version",
```

Comment:
The transaction version isn't set in our result at all.  There is no field for it, but it would be useful.

Should this even be the *transaction* version, and not the *script* version?  Or should we include both?

I went to check dcrd to see what it used for this, and it also has this same help text describing a transaction version, but the result similarly has no version field.

cc @davecgh

Reply:
It used to be the transaction version, but that was removed by @rstaudt2 in decred/dcrd@d05271c4b12089dfc8f3691135a49b6f0145eea7.  It looks like the help entry wasn't removed as well though.

Reply:
That aside, I think the RPC should be updated to return a "scriptVersion" field.  This is going to become important when we introduce a new version.  I'll open an issue on dcrd to add that.

Reply:
I see https://github.com/decred/dcrd/pull/2650 is already open to add `scriptPubKey.version` for dcrd. Will need to bump `github.com/decred/dcrd/rpc/jsonrpc/types/v3` to v4/master (after the dcrd pr is merged) to be able to set that value in wallet. Any plans to bump dcrd deps versions for wallet soon @jrick? I can open a PR for that if no one else is available to get on it sooner; I suspect I'll need the deps updated for something else on dcrdex.

Reply:
dep bump is happening soon.  I have a half finished diff sitting in my repo.